pennylane/labs/tests/resource_estimation/ops/qubit/test_qchem_ops.py

```diff
@@ -18,22 +18,6 @@
 # pylint: disable=use-implicit-booleaness-not-comparison,no-self-use
 
 
-# TODO: implement in resource_symbolic_ops branch
-class TestSingleExcitation:
```

Comment:
These should be removed in the base PR? probably need to pull latest changes in here